analyzers/src/ITs.JsonParser/ITs.JsonParser.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.9.1" />
+  </ItemGroup>
+
```

Comment:
You need to create a target here that will copy the binaries into packaging folder. See `SonarAnalyzer.RuleDescriptorGenerator` for inspiration of your theft.

Reply:
Discussed offline, applied the suggestion